dfs/backup.go

```diff
@@ -547,68 +549,6 @@ func (dfs *DistributedFilesystem) loadSnapshots(tenantID, infile string) error {
 	return nil
 }
 
-func (dfs *DistributedFilesystem) importSnapshots(filename string) error {
```

Comment:
It was not used and called dfs.Rollback.  I cleaned it up to help simplify my code analysis.